### PR TITLE
[PBW-3496][VTC] Add OO.VIDEO objects TECHNOLOGY and FEATURE

### PR DESCRIPTION
**Pull request by angieisrani**

### CODE REVIEW

File: js/utils/constants.js [hunk 1]
```diff
@@ -1278,6 +1279,69 @@
...
+         * @description <code>OO.VIDEO.TECHNOLOGY.MIXED ('mixed')</code>:
+         *   The core video technology used may be based on any one of multiple core technologies.
+         * @constant OO.VIDEO.TECHNOLOGY.MIXED
+         * @type {string}
+         */
+        MIXED: "mixed",
+
+        /**
+         * @description <code>OO.VIDEO.TECHNOLOGY.OTHER ('other')</code>:
+         *   The core video technology is based on a unique core video technology.
```
Comment (mityaha) [inline js/utils/constants.js]: "unique core video technology" doesn't correlate well with "OTHER" - isn't it worth elaborating little more on this comment?

Comment (angieisrani) [inline js/utils/constants.js]: Hm, yes i'm conflicted about this.  If someone comes in with a technology that isn't one of the above do we want to create a value like other for them to use or do we always want to create a new constant for them?  By unique I just meant a technology that we wouldn't want to create a constant out of, say it's proprietary.  May be not worth having other at all...

Comment (mityaha) [inline js/utils/constants.js]: Maybe you can say something like, "The core video technology that isn't one of the above" ?

Comment (angieisrani) [inline js/utils/constants.js]: Yes I almost did this and then removed it.  I will put it back to that

**Comment by angieisrani** [comment]
@mityaha Ready for re-review, thanks

**Comment by mityaha** [comment]
Yes, this is better, thanks.

**Pull request merged**